src/Analyser/NodeScopeResolver.php

```diff
@@ -793,11 +799,7 @@ private function lookForEnterVariableAssign(Scope $scope, Expr $node): Scope
 				if ($listItem === null) {
 					continue;
 				}
-				$listItemValue = $listItem;
-				if ($listItemValue instanceof Expr\ArrayItem) {
-					$listItemValue = $listItemValue->value;
```

Comment:
This has been here because the phpDoc lied in some PHP-Parser version but let's hope it doesn't break anything.

Reply:
Yeah I think I remember that.